packages/flutter_tools/test/integration.shard/test_driver.dart

```diff
@@ -702,14 +701,14 @@ class FlutterRunTestDriver extends FlutterTestDriver {
       id: requestId,
       ignoreAppStopEvent: method == 'app.stop',
     );
-    _process.stdin.writeln(jsonEncoded);
+    _process?.stdin.writeln(jsonEncoded);
     final Map<String, dynamic> response = await responseFuture;
 
     if (response['error'] != null || response['result'] == null) {
       _throwErrorResponse('Unexpected error response');
     }
 
-    return response['result'];
+    return response['result'] as int?;
```

Comment:
Here we are getting this error
```type 'bool' is not a subtype of type 'int?' in type cast```

Reply:
`int?` is not the right type here. First on line 707 there is already a null check on `response['result']`. Second in the `_restart` method on line 623, the result of `_sendRequest` is assigned to a `Map<String, dynamic>?`.

This PR needs some more thought.

Reply:
I have make it dynamic.